docs/source/tutorials/sentence_transformers.mdx

```diff
@@ -0,0 +1,84 @@
+limitations under the License.
+-->
+# Sentence Transformers on AWS Inferentia with Optimum Neuron
+
+_There is a notebook version of that tutorial [here](https://github.com/huggingface/optimum-neuron/blob/main/notebooks/sentence-transformers/getting-started.ipynb)._
+
+This guide explains how to convert, load, and use [Sentence Transformers (SBERT)](https://www.sbert.net/) models on AWS Inferentia2 using Optimum Neuron, enabling efficient calculation of embeddings. Sentence Transformers are powerful model for generating sentence embeddings. You can use this Sentence Transformers to compute sentence / text embeddings for more than 100 languages. These embeddings can then be compared e.g. with cosine-similarity to find sentences with a similar meaning. This can be useful for semantic textual similar, semantic search, or paraphrase mining.
+
+_Note: Currently only text models are supported, we are working on vision support for CLIP._
+
+
+## Convert Sentence Transformers model to AWS Inferentia2  
+
+First, you need to convert your Sentence Transformers model to a format compatible with AWS Inferentia2. You can compile Sentence Transformers models with Optimum Neuron using the `optimum-cli` or `NeuronModelForSentenceTransformers` class. Below you will find an example for both approaches. We have to make sure `sentence-transformers` is installed. Thats only needed for exporting the model. 
+
+```bash
+pip install sentence-transformers
+```
+
+Here we will use the `NeuronModelForSentenceTransformers`, which can be used to convert any Sntence Transformers model to a format compatible with AWS Inferentia2 or load already converted models. When exporting models with the `NeuronModelForSentenceTransformers` you need to set `export=True` and define the input shape and batch size. The input shape is defined by the `sequence_length` and the batch size by `batch_size`. 
+
+```python
+from optimum.neuron import NeuronModelForSentenceTransformers
+
+# Sentence Transformers model from HuggingFace
+model_id = "BAAI/bge-small-en-v1.5"
+input_shapes = {"batch_size": 1, "sequence_length": 384}  # mandatory shapes
+
+# Load Transformers model and export it to AWS Inferentia2
+model = NeuronModelForSentenceTransformers.from_pretrained(model_id, export=True, **input_shapes)
+
+# Save model to disk
+model.save_pretrained("bge_emb_inf2/")
+```
+
+Here we will use the `optimum-cli` to convert the model. Similar to the `NeuronModelForSentenceTransformers` we need to define our input shape and batch size. The input shape is defined by the `sequence_length` and the batch size by `batch_size`. The `optimum-cli` will automatically convert the model to a format compatible with AWS Inferentia2 and save it to the specified output directory.
+
+```bash
+!optimum-cli export neuron -m BAAI/bge-small-en-v1.5 --library-name sentence_transformers --sequence_length 384 --batch_size 1 --task feature-extraction bge_emb_inf2/
```

Comment:
```suggestion
optimum-cli export neuron -m BAAI/bge-small-en-v1.5 --library-name sentence_transformers --sequence_length 384 --batch_size 1 --task feature-extraction bge_emb_inf2/
```

since it's not a jupyter notebook